setup.py

```diff
@@ -49,6 +49,7 @@
     "tablib>=0.12.1,<0.13",
     "werkzeug>=0.14.1 ; python_version<'3.10'",
     "werkzeug>=0.15.0 ; python_version>='3.10'",
+    "swagger_spec_validator>=2.4.0,<3.0.0; python_version<'3.7'",  # 3.0.0 drops support for python 3.6
```

Comment:
Perhaps you can move the comment away and and just use the verbose git commit message? (With `closes #nnnn` there.) The commit messages are better suited for recording (possibly long) decisions as to the "why" behind commits, so we can use them homogeneously also for recording (small) reasons.